## Problem Statement/Goal
The team wants to find the month they're likely to contact the most clients, so they can schedule a product upgrade announcement. Which month does the team tend to contact the greatest percentage of its clients?

(In addition to providing an answer, please either include a public link to a spreadsheet showing your work, or describe your process for answering the question.)

## Imports

In [3]:
import pandas as pd
import matplotlib.pyplot as plt
import seaborn as sns
import numpy as np
%matplotlib inline

## Data Collection

In [5]:
df = pd.read_csv('./Data.csv')

## Initial Viewing of Data

In [6]:
df.head()

Account manager                 Client Name Date of Contact
0     Milan Crona     Wyman, Farrell and Haag      2015-10-31
1    Aidan Pouros  Veum, McClure and Schuster      2013-12-15
2    Aidan Pouros  Veum, McClure and Schuster      2013-12-07
3     Milan Crona             Armstrong Group      2013-11-02
4    Aidan Pouros   Lueilwitz, Moore and Hahn      2013-12-25

## Cleaning

This code may lack finesse and could break if this data wasn't so clean, but it works for the purposes I have at the moment.

In [7]:
def map_function(row):
    return row[0:4]
df['year'] = df['Date of Contact'].map(lambda x: map_function(x))

In [8]:
def map_function(row):
    return row[8:]
df['day'] = df['Date of Contact'].map(lambda x: map_function(x))

In [9]:
def map_function(row):
    return row[5:7]
df['month'] = df['Date of Contact'].map(lambda x: map_function(x))

## Exploritory Data Analysis

In [7]:
df['month'].value_counts()

10    213
09    121
08     79
06     77
12     77
05     69
07     66
11     65
02     64
01     60
03     58
04     52
Name: month, dtype: int64

# Answer:
The most calls have been made in October with 213 over the years on reccord, with September being in second place at 121 calls, total.